### PR TITLE
RELATED: RAIL-2093 Remove unnecessary catalog calls

### PR DESCRIPTION
**Pull request by kandl**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-backend-bear/src/backend/workspace/catalog/factory.ts [hunk 1]
```diff
@@ -320,17 +329,22 @@ export class BearWorkspaceCatalogFactory implements IWorkspaceCatalogFactory {
             return [attr.links.defaultDisplayForm, ...geoPins];
         });
         const dateAttributeUris = dateDatasetAttributes.map((attr) => attr.attributeMeta.uri);
-        const dateDisplayFormUris = dateDatasetAttributes.map((attr) => attr.defaultDisplayFormMeta.uri);
 
         return this.authCall((sdk) =>
             sdk.md.getObjects<BearDisplayFormOrAttribute>(
                 this.workspace,
-                uniq([...attributeUris, ...displayFormUris, ...dateAttributeUris, ...dateDisplayFormUris]),
```
Comment (kandl) [inline libs/sdk-backend-bear/src/backend/workspace/catalog/factory.ts:328]: dateDisplayForms are not used anywhere

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-backend-bear/tests/wiremock/recordings/mappings/gdc_account_profile_889bd5b237cc5af3e6c4749bdd551a8d-f25a9935-cd13-4e3f-b17e-41ee440cec2f.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            },
+            "Content-Type": {
+                "equalTo": "application/json",
+                "caseInsensitive": true
+            }
+        }
+    },
+    "response": {
+        "status": 200,
+        "body": "{\"accountSetting\":{\"links\":{\"self\":\"/gdc/account/profile/889bd5b237cc5af3e6c4749bdd551a8d\",\"projects\":\"/gdc/account/profile/889bd5b237cc5af3e6c4749bdd551a8d/projects\",\"auditEvents\":\"/gdc/account/profile/889bd5b237cc5af3e6c4749bdd551a8d/auditEvents\",\"domain\":\"/gdc/domains/default\"},\"firstName\":\"Matyáš\",\"lastName\":\"Kandl\",\"companyName\":\"GoodData\",\"position\":null,\"created\":\"2018-02-27 14:41:22\",\"updated\":\"2019-09-30 14:30:10\",\"country\":null,\"phoneNumber\":\"777545314\",\"authenticationModes\":[],\"login\":\"matyas.kandl@gooddata.com\",\"timezone\":null,\"ssoProvider\":null,\"email\":\"matyas.kandl@gooddata.com\",\"language\":\"en-US\",\"ipWhitelist\":null,\"effectiveIpWhitelist\":null}}",
```
Comment (no23reason) [inline libs/sdk-backend-bear/tests/wiremock/recordings/mappings/gdc_account_profile_889bd5b237cc5af3e6c4749bdd551a8d-f25a9935-cd13-4e3f-b17e-41ee440cec2f.json:19]: You may want to remove your phone number from here, fill it with whatever

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/271/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/527/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2052/ : SUCCESS in 9m 40s

**Pull request merged**